libqtile/widget/clock.py

```diff
@@ -88,3 +101,30 @@ def poll(self):
         else:
             now = datetime.now(timezone.utc).astimezone()
         return (now + self.DELTA).strftime(self.format)
+
+    @expose_command
+    def update_timezone(self, timezone: str | tzinfo | None = None):
+        """
+        Force the clock to update timezone information.
+
+        If the method is called with no arguments then the widget will reload
+        the timzeone set on the computer (e.g. via ``timedatectl set-timezone ..``).
+        This will have no effect if you have previously set a ``timezone`` value.
+
+        Alternatively, you can pass a timezone string (e.g. ``"Europe/Lisbon"``) to change
+        the specified timezone. Setting this to an empty string will cause the clock
+        to rely on the system timezone.
+        """
+        if timezone:
+            self._check_timezone(timezone)
```

Comment:
minor, but i wonder if this would look cleaner if timezone returned the right value, so these four lines would be `self.timezone = _lift_timezone(timezone)` or similar.

Reply:
Yes.